spec/unit/property/list_spec.rb

```diff
@@ -32,11 +32,11 @@
     end
 
     it "should return the passed in array values joined with the delimiter from is_to_s" do
-      expect(@property.is_to_s(["foo","bar"])).to eq("foo,bar")
+      expect(@property.is_to_s(['foo','bar'])).to eq("['foo', 'bar']")
     end
 
-    it "should be able to correctly convert ':absent' to a string" do
-      expect(@property.is_to_s(:absent)).to eq("absent")
```

Comment:
The `present` and `absent` bare words come from `ensurable`, but the list is defined by the type's parameter/properties, e.g. `service => running`. You can programmatically get this information via:

```ruby
irb(main):005:0> puts Puppet::Type.type(:service).attrclass(:ensure).value_collection.values
stopped
running
```